lib/node_modules/@stdlib/lapack/base/dlacpy/README.md

```diff
@@ -0,0 +1,252 @@
+
+<!-- eslint-disable stdlib/capitalized-comments -->
+
+```javascript
+var Float64Array = require( '@stdlib/array/float64' );
+
+// Initial arrays...
+var A0 = new Float64Array( [ 1.0, 2.0, 3.0, 4.0, 5.0 ] );
+var B0 = new Float64Array( 5 );
+
+// Create offset views...
+var A1 = new Float64Array( A0.buffer, A0.BYTES_PER_ELEMENT*1 ); // start at 1st element
+var B1 = new Float64Array( B0.buffer, B0.BYTES_PER_ELEMENT*1 ); // start at 1st element
+
+dlacpy( 'row-major', 'all', 2, 2, A1, 2, B1, 2 );
+// B0 => <Float64Array>[ 0.0, 2.0, 3.0, 4.0, 5.0 ]
+```
+
+#### dlacpy.ndarray( order, uplo, M, N, A, LDA, offsetA, B, LDB, offsetB )
+
+Copies all or part of a two-dimensional matrix `A` to another matrix `B` using alternative indexing semantics.
+
+```javascript
+var Float64Array = require( '@stdlib/array/float64' );
+
+var A = new Float64Array( [ 0.0, 1.0, 2.0, 3.0, 4.0 ] );
+var B = new Float64Array( [ 0.0, 0.0, 11.0, 312.0, 53.0, 412.0 ] );
+
+dlacpy.ndarray( 'row-major', 'all', 2, 2, A, 2, 1, 1, B, 2, 1, 2 );
+// B => <Float64Array>[ 0.0, 0.0, 1.0, 2.0, 3.0, 4.0 ]
+```
+
+The function has the following parameters:
+
+-   **order**: storage layout.
+-   **uplo**: specifies whether the upper or lower triangular part of the symmetric matrix `A` is supplied.
+-   **M**: number of rows in `A`.
+-   **N**: number of columns in `A`.
+-   **A**: input [`Float64Array`][mdn-float64array].
+-   **strideA1**: stride of the first dimension of `A`
+-   **strideA2**: stride of the second dimension of `A`
+-   **offsetA**: starting index for `A`.
+-   **B**: output [`Float64Array`][mdn-float64array].
+-   **strideB1**: stride of the first dimension of `B`
+-   **strideB2**: stride of the second dimension of `B`
+-   **offsetB**: starting index for `B`.
+
+While [`typed array`][mdn-typed-array] views mandate a view offset based on the underlying buffer, the offset parameters support indexing semantics based on starting indices. For example,
+
+```javascript
+var Float64Array = require( '@stdlib/array/float64' );
+
+var A = new Float64Array( [ 0.0, 1.0, 2.0, 3.0, 4.0 ] );
+var B = new Float64Array( [ 0.0, 0.0, 11.0, 312.0, 53.0, 412.0 ] );
+
+dlacpy.ndarray( 'row-major', 'all', 2, 2, A, 2, 1, 1, B, 2, 1, 2 );
+// B => <Float64Array>[ 0.0, 0.0, 1.0, 2.0, 3.0, 4.0 ]
+```
+
+</section>
+
+<!-- /.usage -->
+
+<section class="notes">
+
+## Notes
+
+-   `dlacpy()` corresponds to the [LAPACK][lapack] routine [`dlacpy`][dlacpy].
+
+</section>
+
+<!-- /.notes -->
+
+<section class="examples">
+
+## Examples
+
+<!-- eslint no-undef: "error" -->
+
+```javascript
+var uniform = require( '@stdlib/random/array/uniform' );
+var dlacpy = require( '@stdlib/lapack/base/dlacpy' );
+
+var M = 5;
+var N = 8;
+
+var opts = {
+    'dtype': 'float64'
+};
+
+var A = uniform( M * N, -10.0, 10.0, opts );
+var B = uniform( M * N, -10.0, 10.0, opts );
+
+dlacpy( 'row-major', 'all', M, N, A, N, B, N );
+console.log( B );
```

Comment:
You want to print the before and after. Just printing `B` after performing the operation doesn't really help the user get insight into what is happening.

Perhaps a more interesting example would be if `B` was an array of zeros. And then you copied the upper triangular part of `A` to `B`. Then it would be easier to see which values were copied over (i.e., all the non-zero values!). And to make things a bit cleaner you can use `@stdlib/random/array/discrete-uniform` to generate values between 10 and 20. This would make the printed output visually tidier.